src/main/java/seedu/address/model/consultation/Consultation.java

```diff
@@ -70,7 +69,8 @@ public boolean isSameConsultation(Consultation consultation) {
         return consultation != null
                 && consultation.getDate().equals(getDate())
                 && consultation.getTime().equals(getTime())
-                && consultation.getName().equals(getName());
+                && (consultation.getType().type.equals(ConsultationType.PERSONAL)
```

Comment:
Do we only support PERSONAL type consults? Seems like this line should be moved to below. Since it is considered as "one other identity field"

Reply:
GROUP consults can be together at the same time, but PERSONAL consults cannot be at the same time right, hence for the check of whether they are PERSONAL or whether the consults have the same name.

Reply:
Tracked in https://github.com/AY2021S1-CS2103T-F11-1/tp/issues/178